WTRSimlib/src/main/java/simulation/launch/UnityVerifier.java

```diff
@@ -0,0 +1,79 @@
+package simulation.launch;
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetAddress;
+import java.net.SocketException;
+
+/**
+ * IoVerifier is responsible for verifying 
+ * that Input/Output with Unity is working, 
+ * and by extension confirms that Unity itself 
+ * has launched and is sending/receiving packets.
+*/
+public class UnityVerifier implements Runnable {
+
+    int timeOutMili = 2000;
+    int verifTimeOutMili = 60000;
+    int clientPort = 4513;
+    DatagramSocket sendSocket;
+    DatagramSocket receiveSocket;
+    SocketHandler socketHandler;
+
+    public boolean kill = false;
+
+    UnityVerifier(DatagramSocket sendSocket, DatagramSocket 
+        receiveSocket, SocketHandler socketHandler) {
```

Comment:
return on each comma and on the opening parentheses

Reply:
will make this change